Cubical/Categories/Enriched/Presheaf.agda

```diff
@@ -0,0 +1,368 @@
+      𝓓[ reindPsh F EC[ x , y ] ⊗𝓓 reindPsh F EC[ y , z ] ,
+         reindPsh F (EC[ x , y ] ⊗𝓒 EC[ y , z ]) ]
+    distrib = natTrans (λ _ x → x) λ _ → refl
+
+    Eid : {x : ob EC} → 𝓓[ unit𝓓 , reindPsh F EC[ x , x ] ]
+    Eid = const ●ᵛ (EC .id ∘ˡ (F ^opF))
+
+    Eseq : {x y z : ob EC} →
+      𝓓[ reindPsh F EC[ x , y ] ⊗𝓓 reindPsh F EC[ y , z ] ,
+         reindPsh F EC[ x , z ] ]
+    Eseq {x}{y}{z} = distrib ●ᵛ (EC .seq x y z ∘ˡ (F ^opF))
+
+    BaseChange : EnrichedCategory 𝓓Mon ℓE
+    BaseChange .ob = ob EC
+    BaseChange .Hom[_,_] c c' = reindPsh F (EC[ c , c' ])
+    BaseChange .id {x} = Eid
+    BaseChange .seq x y z = Eseq
+    BaseChange .⋆IdL x y =
+      makeNatTransPath (funExt λ d → funExt⁻
+        (cong (N-ob) (EC .⋆IdL x y)) (F-ob F d))
+    BaseChange .⋆IdR x y =
+      makeNatTransPath (funExt λ d → funExt⁻
+        (cong (N-ob) (EC .⋆IdR x y)) (F-ob F d))
+    BaseChange .⋆Assoc x y z w =
+      makeNatTransPath (funExt λ d → funExt⁻
+        (cong N-ob (EC .⋆Assoc x y z w)) (F-ob F d))
+
+
+  -- construct a normal category from an enriched category
+  module _ {C : Category ℓ ℓ'}{ℓS : Level}  where
+    open model C {ℓS}
+
+    module _ (E : EnrichedCategory 𝓟Mon ℓS ) where
+
+      open EnrichedCategory E renaming (ob to obE ; Hom[_,_] to E[_,_];
+        id to idE ; seq to seqE ; ⋆IdL to ⋆IdLE ; ⋆IdR to ⋆IdRE ;
+        ⋆Assoc to ⋆AssocE)
+      open MonoidalCategory 𝓟Mon renaming (_⊗ₕ_ to _⊗ₕ𝓟_ ; η⟨_⟩  to η𝓟⟨_⟩ )
+
+      norm : Category _ _
+      norm .ob = obE
+      norm .Hom[_,_] e₁ e₂ = 𝓟 [ 𝟙 , E[ e₁ , e₂ ] ]
+      norm .id = idE
+      norm ._⋆_ {e₁}{e₂}{e₃} f g = dup ⋆⟨ 𝓟 ⟩ f ⊗ₕ𝓟 g ⋆⟨ 𝓟 ⟩ seqE e₁ e₂ e₃
+      norm .⋆IdL {e₁}{e₂} f =
+          makeNatTransPath (funExt λ c → funExt λ {tt* →
+          λ i → sym (⋆IdLE e₁ e₂) i .N-ob c (tt* , f .N-ob c tt*)})
+      norm .⋆IdR {e₁}{e₂} f =
+        makeNatTransPath (funExt λ c → funExt λ {tt* →
+          λ i → sym (⋆IdRE e₁ e₂) i .N-ob c (f .N-ob c tt* ,  tt*)})
+      norm .⋆Assoc f g h =
+        makeNatTransPath (funExt λ c → funExt λ{ tt* →
+          λ i → ⋆AssocE _ _ _ _ i .N-ob c
+            (f .N-ob c tt* , (g .N-ob c tt* , h .N-ob c tt*))})
+      norm .isSetHom = 𝓟 .isSetHom
+
+  -- Psh(SET) enriched category from a normal category
+  module _ (C : Category ℓ ℓ') where
+    open model (SET ℓ') {ℓ'} renaming (𝓟Mon to 𝓟Set)
+    open MonoidalCategory 𝓟Set renaming (Hom[_,_] to 𝓟Set[_,_] ; _⊗_ to _⊗𝓟_ ;
+      η⟨_⟩ to η𝓟⟨_⟩ ; ρ⟨_⟩ to ρ𝓟⟨_⟩ ; _⊗ₕ_ to _⊗ₕ𝓟_) hiding (C)
+
+    -- set indexed hom
+    iHom : (c c' : ob C) → ob 𝓟
+    iHom c c' = LiftF ∘F ((SET _) [-, (C [ c , c' ]) , C .isSetHom ])
+
+    id' : {c : ob C} → 𝓟 [ 𝟙 , iHom c c ]
+    id' {c} .N-ob _ tt* = lift (λ _ → C .id)
+    id' {c} .N-hom _ = refl
+
+    seqE : (c₁ c₂ c₃ : ob C) → 𝓟 [ iHom c₁ c₂ ⊗𝓟 iHom c₂ c₃ , iHom c₁ c₃ ]
+    seqE c₁ c₂ c₃ .N-ob A (f , g) = lift λ a → f .lower a ⋆⟨ C ⟩ g .lower a
+    seqE c₁ c₂ c₃ .N-hom _ = refl
+
+    enrich : EnrichedCategory 𝓟Set ℓ
```

Comment:
This construction should go in `Cubical.Categories.Enriched.Instances.FromCategory`